admin_ui/src/components/RowFormSearch.vue

```diff
@@ -59,23 +66,21 @@
                     v-bind:title="property.title"
                     v-bind:type="property.type || property.anyOf[0].type"
                     v-bind:value="getValue(property.title)"
-                    v-bind:choices="property.extra.choices"
                 />
             </template>
         </div>
     </div>
 </template>
 
 <script lang="ts">
-import Vue from "vue"
-
+import { defineComponent } from "vue"
 import KeySearch from "./KeySearch.vue"
 import InputField from "./InputField.vue"
 
-export default Vue.extend({
+export default defineComponent({
     props: {
         row: Object,
-        schema: Object,
+        schema: Object as any,
```

Comment:
Vue 3 has a thing called `PropType`. Here we might want to try:

```
schema: Object as PropType<object>
```

Reply:
Unfortunately does not work. I tried `schema: Object as PropType<object>` but then `this.schema?.required` error `Property 'required' does not exist on type 'object'`.

Reply:
OK, no problem. With a lot of these type issues we'll be able to solve them in the future by using the composition API more.